### ISSUE TITLE
Add icons from https://tabler-icons.io

### ISSUE THREAD

**Issue by Insire** [MahApps/MahApps.Metro.IconPacks#255]
**Is your feature request related to a problem? Please describe.**
no.

**Describe the solution you'd like**
A new icon pack to be added, that includes the icons from https://tabler-icons.io/

**Describe alternatives you've considered**
none really, just think that the icons are neat.

**Additional context**
-

**Comment by punker76** [comment]
Yes, the icons are neat, but it's not so easy to get the needed paths for this lib. I can add the outlined svg with some effort, maybe later...
